src/game/enemy/systems.rs

```diff
@@ -82,74 +121,132 @@ fn spawn_health_bar(builder: &mut ChildBuilder) {
     asset_server: Res<AssetServer>,
-    mut texture_atlases: ResMut<Assets<TextureAtlas>>,
+    enemy_spawn_timer: Res<EnemySpawnTimer>,
+    enemy_handles: Res<EnemyHandles>,
+    mut word_bank: ResMut<WordBank>,
+    word_q: Query<&Word, (With<Word>, Without<InputText>)>,
 ) {
-    let win = win_q.get_single().unwrap();
-
-    let words = vec!["cat", "rat", "delicious", "delete", "banana"];
-
-    for (i, w) in words.iter().enumerate() {
-        let cur_sprite = SOLDIER_01_RUN;
-
-        let texture_handle = asset_server.load(cur_sprite.path);
-        let texture_atlas = TextureAtlas::from_grid(
-            texture_handle,
-            Vec2::new(cur_sprite.x, cur_sprite.y),
-            cur_sprite.cols,
-            cur_sprite.rows,
-            None,
-            None,
-        );
-        let texture_atlas_handle = texture_atlases.add(texture_atlas);
-        let animation_indices = AnimationIndices { first: 0, last: 6 };
-        let x_offset = i * 130;
-        let rotate = if i % 2 == 0 {
-            Quat::from_rotation_y(std::f32::consts::PI)
+    if enemy_spawn_timer.timer.finished() {
+        println!("<< Spawning enemy from timer >>");
+        let win = win_q.get_single().unwrap();
+        let font = asset_server.load("fonts/fyodor/truetype/Fyodor-BoldCondensed.ttf");
+        let random_x = if random::<f32>() < 0.5 {
+            random::<f32>() * win.width() / 2.
         } else {
-            Quat::default()
+            -random::<f32>() * win.width() / 2.
         };
-        let transform = Transform::from_xyz(
-            win.width() / 4.0 + (x_offset as f32),
-            win.height() / 2.0 - 130.0,
-            1.0,
-        )
-        .with_rotation(rotate);
-
-        commands.spawn((
-            SpriteSheetBundle {
-                texture_atlas: texture_atlas_handle,
-                sprite: TextureAtlasSprite::new(0),
-                transform,
-                ..default()
-            },
-            animation_indices,
-            AnimationTimer(Timer::from_seconds(0.1, TimerMode::Repeating)),
-            AnimateSprite,
-            Enemy {
-                ..default()
-            },
-            Word(WordTarget::Enemy(3), w.to_string()), // for testing word uniqueness
-        ));
-        println!("Spawned enemy: {w}.");
+        let random_y = if random::<f32>() < 0.5 {
+            random::<f32>() * win.height() / 2.
+        } else {
+            -random::<f32>() * win.height() / 2.
+        };
+        let animation_indices = EnemyAnimations::SovietIdle.get_indices();
+        let mut rng = thread_rng();
+        commands
+            .spawn((
+                SpriteSheetBundle {
+                    texture_atlas: enemy_handles.soviet_idle.clone(),
+                    sprite: TextureAtlasSprite::new(
+                        rng.gen_range(animation_indices.0..animation_indices.1),
+                    ),
+                    transform: Transform::from_xyz(random_x, random_y, 1.),
```

Comment:
think we could take this and the initial spawn and put it in its own function to avoid repeated code. Think the only things changed right now is the transform coordinates